src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisProvider.java

```diff
@@ -82,7 +82,11 @@ public CompletableFuture<SecurityAnalysisReport> run(Network network,
 
         AbstractSecurityAnalysis<?, ?, ?, ?, ?> securityAnalysis;
         if (loadFlowParameters.isDc()) {
-            securityAnalysis = new DcSecurityAnalysis(network, matrixFactory, selectedConnectivityFactory, runParameters.getMonitors(), runParameters.getReportNode());
+            if (OpenSecurityAnalysisParameters.getOrDefault(runParameters.getSecurityAnalysisParameters()).isDcFastMode()) {
```

Comment:
@obrix @jeandemanged can you check this line about security analysis parameters please ! I am not familiar with them.

Reply:
To be fixed in another PR.